components/base/select/Select.tsx

```diff
@@ -27,7 +27,7 @@ const DefaultDropdownIndicator = (props: any) => {
 const CodeDropdownIndicator = (props: any) => {
   return (
     <components.DropdownIndicator {...props}>
-      <DoubleArrowSelectIcon />
+      <DropdownArrowDouble width={7} />
```

Comment:
Note that the width (and height) props are applied to the `<svg>` tag when everything is rendered